tests/test_yaml_generators/test_yaml_oci_generator.py

```diff
@@ -65,13 +66,13 @@ def test_generate_tags(self, mock_choice):
         """Test label string generator."""
 
         dc = self.yg.default_config()
-        for key in self.module.TAG_KEYS.keys():
+        for key in OCIYamlConstants.tag_keys.keys():
```

Comment:
The `dict_keys` view object isn't needed here. Iterating over the dict will iterate through the keys.
```suggestion
        for key in OCIYamlConstants.tag_keys:
```